docs/vue/platform.md

```diff
@@ -44,3 +44,29 @@ Below is a table listing all the possible platform values along with correspondi
 | phablet       | a phablet device                         |
 | pwa           | a PWA app                                |
 | tablet        | a tablet device                          |
+
+## Customizing Platform Detection Methods
+
+The method used to detect a specific platform can also be overridden by providing an alternative method in the global Ionic [config](/docs/vue/config). Each method takes `window` as a parameter and returns a boolean. 
```

Comment:
Minor detail, but these are functions as there is no class/object associated with each function call.

```suggestion
The functions used to detect a specific platform can also be overridden by providing alternative functions in the global Ionic [config](/docs/vue/config). Each method takes `window` as a parameter and returns a boolean. 
```